webapp/src/components/admin_settings/custom_attribute_settings.jsx

```diff
@@ -118,6 +126,19 @@ export default class CustomAttributesSettings extends React.Component {
                         id={this.state.attributes.size}
                     />
                 </div>
+                <ConfirmModal
+                    show={this.state.showDeleteModal}
+                    title={'Delete Attribute'}
+                    message={
+                        'Are you sure you want to remove the attribute : "' +
```

Comment:
NIT: remove the space before the colon. (i.e. `... the attribute: ` instead of `... the attribute :`)

Reply:
Done :)